### PR TITLE
Adding github actions

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -8,3 +8,14 @@ jobs:
...
+                  args: analyze
+    php-cs-fixer:
+        name: PHP-CS-Fixer
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@master
+            - name: PHP-CS-Fixer
+              uses: docker://oskarstark/php-cs-fixer-ga
+              with:
+                args: --config=.project.php_cs --diff --dry-run
```
Comment (norkunas) [inline .github/workflows/main.yml]: `project.php_cs` should probably be `.php_cs` ? :slightly_smiling_face:

Comment (Nyholm) [reply .github/workflows/main.yml]: Yeah, I just copied code. =)

**Comment by Nyholm** [comment]
Im done testing now.

**Comment by norkunas** [comment]
Thank you @Nyholm !

**Pull request merged**